packages/wrangler/src/reporting.ts

```diff
@@ -43,12 +43,22 @@ export function initReporting() {
 
 async function appendReportingDecision(userInput: "true" | "false") {
   const homePath = path.join(os.homedir(), ".wrangler/config/");
-  fs.mkdirSync(homePath, { recursive: true });
-  await appendFile(
-    path.join(homePath, "reporting.toml"),
-    `error_tracking_opt = ${userInput} # Sentry \nerror_tracking_opt_date = ${new Date().toISOString()} # Sentry Date Decision \n`,
-    { encoding: "utf-8" }
-  );
+  const reportingConfigLocation = path.join(homePath, "reporting.toml");
+  try {
+    if (!fs.existsSync(reportingConfigLocation)) {
+      fs.mkdirSync(homePath, { recursive: true });
+    }
+    await writeFile(
```

Comment:
Good call to change this to `writeFile()` - I think it was originally `appendFile()` when we were writing it into the `default.toml` file, and we didn't want to overwrite what was already there.